keyboards/tweetydabird/lotus58/nanoboot/keymaps/default/info.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
`info.json` files at the keymap level wont work, so none of this configuration is being picked up.